src/model/liderbord.ts

```diff
@@ -0,0 +1,10 @@
+import Resource from "./resource";
+interface Liderbord {
+  // Identifier of the liderbord (key)
+  id: string;
+  title: string;
+  tags: string[];
+  resources: Resource[];
+}
```

Comment:
You gotta add the following: 
- topic : string;
AS OPTIONAL BUT TO SIMPLIFY :
- nbVotes : number;
- nbResources : number;